stakemachine/__main__.py

```diff
@@ -68,12 +66,27 @@ def main() :
         for name in config["bots"]:
             config["bots"][name]["markets"] = markets
 
+    if (("wif" not in config or not config["wif"]) and "wif" in os.environ):
+        wif = os.environ["wif"]
+        config["wif"] = wif
+
     if (("wallet_host" not in config or not config["wallet_host"]) and
-            ("wif" not in config or not config["wif"])):
+            ("account" not in config or not config["account"])):
         raise Exception(
-            "Need either a wif key or connection details for to the cli wallet."
+            "Need either an account (name) or connection details for to the cli wallet."
```

Comment:
Not sure what you are doing here. The bot either needs a cli wallet or a wif key.
The account name can be obtained from the wif key by asking the network.

I would prefer to have it like this:

* No account given and No wif given: FAIL
* No wif given but an unregistered account given: register account, print wif_key, then terminate (heroku doesn't store 'config' across relaunches)
* wif given for an existing account: go ahead
* wif given with no account associated with it: register random name account, go ahread